tests/datastreams/test_processor.py

```diff
@@ -1,6 +1,8 @@
 import time
```

Comment:
Can we add test coverage for the dsm signal handlers? 

Here are some scenarios we could test:
 - a signal handler already exists
 - A process generating kafka events forks
 - A process generating kafka events is killed gracefully
 - A process generating kafka events is killed abruptly